src/pkg/notifier/notification_handler.go

```diff
@@ -1,7 +1,6 @@
 package notifier
 
 import "context"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380